thunder/distributed/tensor_parallel/column_wise.py

```diff
@@ -239,13 +249,25 @@ def forward(self, tokens: torch.Tensor) -> torch.Tensor:
         for name, p in mod.named_parameters(recurse=False):
             chunked_param_name_to_layer_type["t_" + f"{target_mod_name}.{name}".replace(".", "_")] = type(mod)
 
-    import copy
-
     # Modify module
     for module_name, _ in thunder_module._model.named_modules():
+        submodule = thunder_module.get_submodule(module_name)
```

Comment:
We should share the meta handling eventually (and I wonder if after #564 if we might move materialization to after the transforms), but this is for another PR.